internal/httpserve/handlers/ent_test.go

```diff
@@ -16,74 +17,100 @@ func TestCreateUserInput(t *testing.T) {
 	email := "ww@datum.net"
 	firstName := "Walter"
 	lastName := "White"
+	image := gofakeit.URL()
 
 	testCases := []struct {
 		testName string
 		name     string
 		email    string
 		provider enums.AuthProvider
 		expected ent.CreateUserInput
+		image    string
 	}{
 		{
 			testName: "oauth provider - github",
 			name:     name,
 			email:    email,
 			provider: enums.AuthProviderGitHub,
 			expected: ent.CreateUserInput{
-				FirstName:    &firstName,
-				LastName:     &lastName,
-				Email:        email,
-				AuthProvider: &enums.AuthProviderGitHub,
-				LastSeen:     lo.ToPtr(time.Now().UTC()),
+				FirstName:       &firstName,
+				LastName:        &lastName,
+				Email:           email,
+				AuthProvider:    &enums.AuthProviderGitHub,
+				LastSeen:        lo.ToPtr(time.Now().UTC()),
+				AvatarRemoteURL: &image,
 			},
+			image: image,
 		},
 		{
 			testName: "oauth provider - github, only first name",
 			name:     "meow",
 			email:    email,
 			provider: enums.AuthProviderGitHub,
 			expected: ent.CreateUserInput{
-				FirstName:    lo.ToPtr("meow"),
-				Email:        email,
-				AuthProvider: &enums.AuthProviderGitHub,
-				LastSeen:     lo.ToPtr(time.Now().UTC()),
+				FirstName:       lo.ToPtr("meow"),
+				Email:           email,
+				AuthProvider:    &enums.AuthProviderGitHub,
+				LastSeen:        lo.ToPtr(time.Now().UTC()),
+				AvatarRemoteURL: &image,
 			},
+			image: image,
 		},
 		{
 			testName: "oauth provider - google",
 			name:     name,
 			email:    email,
 			provider: enums.AuthProviderGoogle,
 			expected: ent.CreateUserInput{
-				FirstName:    &firstName,
-				LastName:     &lastName,
-				Email:        email,
-				AuthProvider: &enums.AuthProviderGoogle,
-				LastSeen:     lo.ToPtr(time.Now().UTC()),
+				FirstName:       &firstName,
+				LastName:        &lastName,
+				Email:           email,
+				AuthProvider:    &enums.AuthProviderGoogle,
+				LastSeen:        lo.ToPtr(time.Now().UTC()),
+				AvatarRemoteURL: &image,
 			},
+			image: image,
 		},
 		{
 			testName: "webauthn provider",
 			name:     name,
 			email:    email,
 			provider: enums.AuthProviderWebauthn,
 			expected: ent.CreateUserInput{
-				FirstName:    &firstName,
-				LastName:     &lastName,
-				Email:        email,
-				AuthProvider: &enums.AuthProviderWebauthn,
-				LastSeen:     lo.ToPtr(time.Now().UTC()),
+				FirstName:       &firstName,
+				LastName:        &lastName,
+				Email:           email,
+				AuthProvider:    &enums.AuthProviderWebauthn,
+				LastSeen:        lo.ToPtr(time.Now().UTC()),
+				AvatarRemoteURL: &image,
 			},
+			image: image,
```

Comment:
webauthn provider would never have an image, so this should check with an empty string
```suggestion
			image: image,
```